[2025-04-05T16:41:21.725+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-05T16:41:21.762+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio manual__2025-04-05T16:41:17.071970+00:00 [queued]>
[2025-04-05T16:41:21.771+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_pipeline.upload_to_minio manual__2025-04-05T16:41:17.071970+00:00 [queued]>
[2025-04-05T16:41:21.773+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-05T16:41:21.790+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_to_minio> on 2025-04-05 16:41:17.071970+00:00
[2025-04-05T16:41:21.798+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=320) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-05T16:41:21.798+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'elt_pipeline', 'upload_to_minio', 'manual__2025-04-05T16:41:17.071970+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/elt_pipeline.py', '--cfg-path', '/tmp/tmpqte6keye']
[2025-04-05T16:41:21.799+0000] {standard_task_runner.py:63} INFO - Started process 332 to run task
[2025-04-05T16:41:21.800+0000] {standard_task_runner.py:91} INFO - Job 9: Subtask upload_to_minio
[2025-04-05T16:41:21.823+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-05T16:41:21.865+0000] {task_command.py:426} INFO - Running <TaskInstance: elt_pipeline.upload_to_minio manual__2025-04-05T16:41:17.071970+00:00 [running]> on host 658edd075563
[2025-04-05T16:41:21.971+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt_pipeline' AIRFLOW_CTX_TASK_ID='upload_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-04-05T16:41:17.071970+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-05T16:41:17.071970+00:00'
[2025-04-05T16:41:21.972+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-05T16:41:21.987+0000] {logging_mixin.py:188} INFO - Error uploading file to Minio: [Errno 2] No such file or directory: '/opt/***/bitcoin-2021-01-01.csv'
[2025-04-05T16:41:21.987+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-05T16:41:21.989+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-05T16:41:22.001+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=elt_pipeline, task_id=upload_to_minio, run_id=manual__2025-04-05T16:41:17.071970+00:00, execution_date=20250405T164117, start_date=20250405T164121, end_date=20250405T164122
[2025-04-05T16:41:22.066+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-05T16:41:22.093+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-05T16:41:22.098+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
